config/initializers/inflections.rb

```diff
@@ -25,4 +25,5 @@
   inflect.acronym 'MFL' # Modern foreign languages
   inflect.acronym 'DFE' # Department for Education
   inflect.acronym 'DfE' # Department for Education
+  inflect.acronym 'SCITT'
```

Comment:
```suggestion
  inflect.acronym 'SCITT' # School-Centred Initial Teacher Training
```

Shall we continue the pattern?